### PR TITLE
IDs should be String

### LINKED ISSUE #25: Change type of id field in entities to from Long to String if necessary
Some entities in the "entity" package specify an "id" attribute of type Long. According to the API docs, ids are not guarenteed to be numeric and should have a data type of String instead. Alle entities should quickly be reviewed against API documentation and data type switched if necessary.

### PR DESCRIPTION
**Pull request by bocops**
According to Mastodon API, various ID values are "cast from an integer, but not guaranteed to be a number". This changes entities and related calls from Long to String id values.

Closes #25.

### CODE REVIEW

File: bigbone/src/main/kotlin/social/bigbone/api/method/Reports.kt [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Reports(private val client: MastodonClient) {
      * comment: A comment to associate with the report.
      */
     @Throws(BigboneRequestException::class)
-    fun postReport(accountId: Long, statusId: Long, comment: String): MastodonRequest<Report> {
```
Comment (andregasser) [inline bigbone/src/main/kotlin/social/bigbone/api/method/Reports.kt:39]: statusId should be `String` as well, right?

**Review by andregasser — CHANGES_REQUESTED** [review]
There's one method signature that needs to be changed imho. I have added a comment there.

File: bigbone/src/main/kotlin/social/bigbone/api/method/Reports.kt (hunk 1 above)
Comment (bocops) [reply bigbone/src/main/kotlin/social/bigbone/api/method/Reports.kt:39]: Definitely! I also missed some other, more elusive uses of `Long` ids. :( I hope I now got all of them after doing a full text search on top of everything else. Please re-review.

**Review by andregasser — APPROVED** [review]
Approved!

**Pull request merged**